### PR TITLE
Missing Apps into pipeline

### PR DESCRIPTION
**Pull request by Pawan-MSFT**

### CODE REVIEW

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

File: .github/workflows/build-complete-samples.yml [hunk 1]
```diff
@@ -276,14 +276,6 @@ jobs:
...
               version: '6.0.x'
 
-            - project_path: 'samples/bot-people-picker-adaptive-card/csharp/PeoplePicker/PeoplePicker.csproj'
-              name: 'bot-people-picker-adaptive-card'
-              version: '6.0.x'
-
-            - project_path: 'samples/bot-people-picker-adaptive-card/csharp/PeoplePicker/PeoplePicker.csproj'
-              name: 'bot-people-picker-adaptive-card'
-              version: '6.0.x'
-
```
Comment (Wajeed-msft) [inline .github/workflows/build-complete-samples.yml:286]: Are these duplicates?

**Review by Wajeed-msft — APPROVED** [review]

File: .github/workflows/build-complete-samples.yml (hunk 1 above)
Comment (ChetanSharma-msft) [reply .github/workflows/build-complete-samples.yml:286]: Yes, these are duplicate entries.

**Comment by Pawan-MSFT** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3629035082
Result:
image

**Pull request merged**